.github/workflows/test.yml

```diff
@@ -91,11 +91,22 @@ jobs:
     steps:
       - uses: actions/checkout@v2
       - uses: actions/cache@v2.1.5
+        name: Cache Cargo registry + index
         with:
           path: |
             ~/.cargo/registry
             ~/.cargo/git
           key: ${{ runner.os }}-cargo-${{ hashFiles('**/Cargo.lock') }}
+          restore-keys: |
+            ${{ runner.os }}-cargo-
+      - uses: actions/cache@v2.1.5
+        name: Cache Cargo build
+        with:
+          path: |
+            ~/target/debug
```

Comment:
Is this right? I think you want just `target/debug`.

Reply:
Quite.